File: v2/csharp/src/main/csharp/io/swagger/Model/Sources.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Sources {
    

    
    public List<Source> Sources { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Sources {\n");
      
      sb.Append("  Sources: ").Append(Sources).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/Sources.java
package vestorly.client.model;

import vestorly.client.model.Source;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Sources  {
  
  private List<Source> sources = new ArrayList<Source>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("sources")
  public List<Source> getSources() {
    return sources;
  }
  public void setSources(List<Source> sources) {
    this.sources = sources;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Sources {\n");
    
    sb.append("  sources: ").append(sources).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
